osv/test_tools/test_repository.py

```diff
@@ -0,0 +1,158 @@
+"""test_repository"""
+import pygit2
+import json
+from datetime import datetime
+from enum import Enum
+import os
+import shutil
+import uuid
+
+
+class TestRepository:
+  """ Utilitary class to create a test repository for the git tests
+  """
+
+  class VulnerabilityType(Enum):
+    INTRODUCED = 1
+    FIXED = 2
+    LAST_AFFECTED = 3
+    LIMIT = 4
+    NONE = 5
+
+  _author = pygit2.Signature('John Smith', 'johnSmith@example.com')
+  _commiter = pygit2.Signature('John Smith', 'johnSmith@example.com')
+
+  _initial_commit = None
+
+  def __init__(self, name: str, debug: bool = False):
+    self.debug = debug
+    self.name = name
+    self.introduced = []
+    self.fixed = []
+    self.last_affected = []
+    self.limit = []
+
+    if os.path.exists(f"osv/testdata/test_repositories/{name}"):
+      shutil.rmtree(f"osv/testdata/test_repositories/{name}")
+    self.repo = pygit2.init_repository(
+        f"osv/testdata/test_repositories/{name}", bare=False)
+    #empty initial commit usefull for the creation of the repository
```

Comment:
Fix spelling mistake

```suggestion
    #empty initial commit useful for the creation of the repository
```